.github/workflows/docker-ci.yml

```diff
@@ -143,6 +143,24 @@ jobs:
           docker load --input /tmp/pysages.tar
           docker run --privileged -t pysages bash -c "${SCRIPT}"
 
+  cv-pucker-openmm-unbiased:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: pysages
+          path: /tmp
+      - name: Load and run test
+        run: |
+          docker load --input /tmp/pysages.tar
+          SCRIPT="cd PySAGES/examples/openmm/cv_pucker &&"
+          SCRIPT="${SCRIPT} python unbiased.py" 
+          docker run -t pysages bash -c "cd PySAGES/examples/openmm/cv_pucker && python3 ./unbiased.py"
```

Comment:
you do the work twice here.
you define the SCRIPT variable, but then you also write down the command again.
One is fine.

Reply:
thanks for the code review, I'll fix it